deploy/crds/ocmagent.managed.openshift.io_ocmagents.yaml

```diff
@@ -34,10 +34,55 @@ spec:
           metadata:
             type: object
           spec:
-            description: OCMAgentSpec defines the desired state of OCMAgent
+            description: OcmAgentSpec defines the desired state of OcmAgent
+            properties:
+              ocmAgentConfig:
+                description: OcmAgentConfig defines the configmap name which generated
+                  by operator and consumed by OCM Agent service
+                type: string
+              ocmAgentImage:
+                description: OcmAgentImage defines the image which will be used by
+                  the OCM Agent
+                type: string
+              ocmBaseUrl:
+                description: OcmBaseUrl defines the OCM api endpoint for OCM agent
+                  to access
+                type: string
+              replicas:
+                description: Replicas defines the replica count for the OCM Agent
+                  service
+                format: int32
+                type: integer
+              services:
+                description: Services defines the supported OCM services, eg, service_log,
+                  cluster_management
+                items:
+                  type: string
+                type: array
```

Comment:
For the supported services, is there a naming convention expected like for the Kubernetes services (Refer https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#rfc-1035-label-names)? My query is regarding the underscore "_" character in the names of the services.

Reply:
Or will these be custom strings of supported services mapped internally to OCM services?

Reply:
yes, they are mapping of the services to the OCM.
I just copied them from the api link, like `https://api.openshift.com/api/service_logs/v1`